packages/chain-bridge/src/adapters/StargateAdapter.sol

```diff
@@ -942,7 +764,7 @@ contract StargateAdapter is
         // Decode the compose message
         (
             address recipient,
```

Comment:
this should also use a generic struct instead of  destructuring to unnamed params, so it's easier to maintain across adapters and other parts of the cross chain system